app/views/local_transaction/search.html.erb

```diff
@@ -1,3 +1,7 @@
+<% content_for :extra_headers do %>
+  <%= explore_menu_variant.analytics_meta_tag.html_safe if explore_menu_testable? %>
```

Comment:
You've got a duplicate `content_for :extra_headers` block here. Is there a reason you couldn't put this in the `share/base_page` template?

Reply:
🤦 fixed https://github.com/alphagov/frontend/commit/a7dc4a6c35e3c128eda1de6923842d926ff64fff